clamav-mirror/Dockerfile

```diff
@@ -22,18 +22,13 @@ RUN curl -fsSLO "$SUPERCRONIC_URL" \
  && ln -s "/usr/local/bin/${SUPERCRONIC}" /usr/local/bin/supercronic
 
 
-COPY requirements.txt ./
-
-RUN pip install -r requirements.txt
-
+RUN pip install -vv cvdupdate==1.1.1
```

Comment:
this was the only python dependency used

Reply:
Have you removed the superfluous `requirements.txt` ?

Reply:
removed